feature-group-definitions/flexbox.yml

```diff
@@ -2,13 +2,16 @@ spec: https://drafts.csswg.org/css-flexbox-1/
 caniuse: flexbox
 usage_stats: https://chromestatus.com/metrics/feature/timeline/popularity/1692
 status:
-  is_baseline: true
-  since: "2020-10-20"
+  baseline: low
```

Comment:
(It might be helpful to revisit https://github.com/web-platform-dx/web-features/pull/149#discussion_r1179034596 too.)

`css.properties.flex-basis.{min,max}-content` are the holdouts (appearing in Safari 16). If you drop those two, then the high status prevails.

The top-line summary would look like this:

| Key                                                 | Baseline | Low since date | Versions                                                                                                              |
| :-------------------------------------------------- | :------: | :------------- | --------------------------------------------------------------------------------------------------------------------- |
| **flexbox**                                         |  ✅ High  | 2020-09-22     | Chrome 57<br>Chrome Android 57<br>Edge 79<br>Firefox 81 🔑💎<br>Firefox for Android 81<br>Safari 9<br>Safari on iOS 9 |

Reply:
Leaving my two pence: I was a little surprised to see flexbox as baseline low, however I had no idea/had never checked to see that `flex-basis: min/max-content` were only supported since Safari 16, so perhaps in this case developer expectations are wrong? (Though I'm not sure I've ever used them - and flexbox certainly retains a lot of usefulness without them.)

Longer term, I think we'll need some solution for "the majority of this page is baseline high, but these specific bits are baseline low/not", but for now, perhaps baseline low *is* the appropriate status here? Developers *should* check to see if their use of flexbox (with or without `flex-basis: min/max-content`) supports their target browsers.

Reply:
I agree we'll need a solution for this very common problem, but so far we've mostly handled it by excluding the newer features and punting on how to handle them. I'd prefer to take that path here too, unless we have reason to think the min/max-content are common pitfalls.

Reply:
OK, since we already have a bunch of features of Flexbox commented out, two more won't be that bad. Done with a116ccbbcd2865650151bc50e0db800c71743e64.

I suspect the answer to the display problems is to define related groups, then:

- Design a way to note the existence of "minor" groups on pages that cover higher-level groups at the top of the page
- Show Baseline icons in-line in the body text (e.g., replace/amend the "experimental" beaker with a Baseline icon)
- Update the compat table to provide a list of Baseline statuses that relate to the table